week02/Group Anagrams.cpp

```diff
@@ -0,0 +1,47 @@
+//TC : O(NK) where N is the length of strs, and K is the maximum length of a string in strs
+//SC : O(NK)
+class Solution {
+public:
+    unsigned long long getHash(string &str){
+        unsigned long long h = 1;
+        for(auto c : str){
+            h = (h * (257 + (c - 'a')));
+        }
+        return h;
+    }

```

Comment:
It's better not to use `unsigned long long`. Refer to my [this](https://discord.com/channels/936330694689447978/958964922971217931/970277008183541770) post.